### ISSUE TITLE
Markdown - nested lists

### ISSUE THREAD

**Issue by findthebug** [TimOetting/kirby-builder#147]
https://github.com/getkirby/kirby/issues/2397

Kirby: 3.3
Builder: latest

Markdown - Nested List
The list is just rendered as on level, not nested. The markdown works in a default blueprint field ouside a builder field. 

```Markdown
1. First item
2. Second item
3. Third item
    1. Indented item
    2. Indented item
4. Fourth item
```
```Html
<ol>
<li>First item</li>
<li>Second item</li>
<li>Third item
<ol>
<li>Indented item</li>
<li>Indented item</li>
</ol>
</li>
<li>Fourth item</li>
</ol>
```
https://www.markdownguide.org/basic-syntax

**Comment by TimOetting** [comment]
Hey findthebug,

there is a bug in Kirby that causes indentations from content within Structure Fields and also the Builder Field to be cut off. Unfortunately, I have little influence on this, as this happens in the Kirby core when converting to yaml. There is an issue for tis this, and the fix is planned for Kirby 3.4.0. https://github.com/getkirby/kirby/issues/1308

Sorry that I cannot help you directly with this.

**Comment by findthebug** [comment]
Hey Tim,
Thanks for the answer. No problem, I will test this in the new Rc.
